### PR TITLE
#153009245 configure eslint

### PR DESCRIPTION
**Pull request by oti4me**
#### What does this PR do?

Configure ESLint

#### Description of Task to be completed?

- Install ESLint and other dependencies
- Run eslint --init and choose the airbnb popular style guide
- Configure the generated .eslintrc.yml file to include the airbnb style rules

#### How should this be manually tested?

- Clone online repository
- Navigate to the cloned directory
- Add ESLint as extension to your favourite text editor
- Run npm install
- Open the codebase in your favourite text editor to see linting errors
- Run: npm run eslint on your command prompt to test for linting errors

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

- story type: chore
- story id: 153009245
- story title: Configure ESLint

**Screenshots (if appropriate)**

screen shot 2017-11-21 at 7 16 26 pm

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (temilaj) [inline npm-debug.log]: Why do you we have this file in our repo?

**Review by temilaj — CHANGES_REQUESTED** [review]
I'm curious to know why we're using the .yml configuration file. How well will this work with houndci?

**Comment by oti4me** [comment]
@temilaj I have made the changes you pointed. I have convert the .eslintrc.yml file to .eslintrc file and have remove the npm-debug.log file from the repository.

**Review by temilaj — CHANGES_REQUESTED** [review]
Good work improving your PR. 
However we now have 5 commits that are essentially the same. Take time out to squash these commits into 1, and then raise another PR

**Pull request closed without merging**